catalyst/metrics/_additive.py

```diff
@@ -111,7 +126,7 @@ def on_loader_end(self, runner):
         .. _`minimal examples`: https://github.com/catalyst-team/catalyst#minimal-examples
     """
 
-    def __init__(self, compute_on_call: bool = True):
+    def __init__(self, compute_on_call: bool = True, mode: str = "numpy"):
```

Comment:
could you please add any test to check the functionality? 
simple test case with numpy and torch mertics